docs/cloud-soar/automation.md

```diff
@@ -10,22 +10,24 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 The **Automation** section contains the configuration tools for Cloud SOAR's automation and orchestration features. These tools include Cloud SOAR's Open Integration Framework (OIF), automation rules sets, and playbook editor as well as incoming event details.
 
-To access this section, click the cog icon (<img src={useBaseUrl('img/cloud-soar/cog.png')} alt="cog menu" width="20"/>) > **Automation**.
```

Comment:
It looks like you've eliminated these references to the cog.png image in these docs. If so, I'd also delete the image itself from static/img/cloud-soar/cog.png.

Reply:
Good call, @kimsauce. I made sure the image wasn't being used anywhere, and then I removed it (see commit "[Remove /img/cloud-soar/cog.png](https://github.com/SumoLogic/sumologic-documentation/pull/4041/commits/0b6d3de3cb467026baae448ffb1c360d03dd349d)").